readme.txt

```diff
@@ -127,13 +127,40 @@ Past Contributors: fjarrett, shadyvb, chacha, westonruter, johnregan3, jacobschw
 
 == Upgrade Notice ==
 
-= 4.0.0 =
+= 4.0.1 =
 
 Use only `$_SERVER['REMOTE_ADDR']` as the client IP address for event logs without additional support for `X-Forwarded-For` HTTP request header value which could be spoofed. See the changelog for additional details.
 
 
 == Changelog ==
 
+= 4.0.1 - July 26, 2024 =
+
+- Fix: CLI PHP Type error by [@Soean](https://github.com/Soean) in [#1475](https://github.com/xwp/stream/pull/1475)
+- Fix: [Security] Update `select2` to `4.0.13` (by [@delawski](https://github.com/delawski) in [#1495](https://github.com/xwp/stream/pull/1495))
+- Fix: Uncaught ValueError in Gravity Forms and WordPress SEO connectors by [@tharsheblows](https://github.com/tharsheblows) in [#1508](https://github.com/xwp/stream/pull/1508)

```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore**
Could we add a props to krokodok here? I used his PR as a starting point.